client/components/ellipsis-menu/index.jsx

```diff
@@ -82,7 +84,8 @@ class EllipsisMenu extends Component {
 					disabled={ disabled }
 					className="ellipsis-menu__toggle"
 				>
-					<Gridicon icon="ellipsis" className="ellipsis-menu__toggle-icon" />
+					{ ! icon && <Gridicon icon="ellipsis" className="ellipsis-menu__toggle-icon" /> }
+					{ icon }
```

Comment:
Very nitpicky suggestion to use ternary instead :grin: 

```suggestion
					{ icon ? icon : <Gridicon icon="ellipsis" className="ellipsis-menu__toggle-icon" /> }
```

Reply:
@noahtallen thank you for the review! I like your suggestion, will update the code to use a ternary 👍